usage/app/lib/Config.scala

```diff
@@ -85,7 +85,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
       .withRegion(Regions.EU_WEST_1)
 
   val postfix = if (stage == "DEV") {
-    iamClient.getUser().getUser().getUserName()
+    "b0rked"
```

Comment:
I think that was one of @Reettaphant's changes. Probably best to check with her what's going on here (if this isn't working for you and is the reason you've made this subsititution).